cmds/init/init.go

```diff
@@ -26,6 +26,7 @@ import (
 var (
 	verbose   = flag.Bool("v", false, "print all build commands")
 	ludicrous = flag.Bool("ludicrous", false, "print out information about symlink creation")
+	test = flag.Bool("test", false, "Test mode: don't try to set control tty")
```

Comment:
please run gofmt on this file

(i run gofmt on go files automatically on saving -- it's a useful thing to set up!)